server/routerlicious/packages/services-shared/src/configDumper.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fastRedact from "fast-redact";
+import { ILogger } from "@fluidframework/server-services-core";
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+const errorSanitizationMessage = "FluidREDACTED";
+
+export class ConfigDumper {
+	private readonly config: Record<string, any>;
+	private readonly secretNamesToRedactInConfigDump = [
+		"mongo.globalDbEndpoint",
```

Comment:
instead of hard coding these values, can we fetch a list of names from the config?

If the plan is to add redactedNames in service configs that will call dumpConfig() later, that works too. I want to emphasize that hard coding anything in OSS makes it difficult to update in FRS. So just need to make sure such updates can be done easily in FRS.

Reply:
The config also takes a parameter of additional secret names and concatenates it with the default ones listed here

Reply:
yeah, I saw that. So, every service is going to maintain a list of secretNames to be redacted? I didn't see any list in the service configs.

Reply:
Yeah the plan was to have a redact list for all services which would be the same. So we could add it to the shared_config in FRS. That might be easier to maintain.

Reply:
that's a good idea. we should add that capability. We need a place to manage the list and to make sure that list is editable from FRS (so that we don't have to depend on updating OSS to get the change in FRS)

Reply:
@sharmakhushboo so instead of passing `secretNamesToRedactInConfigDump` as a argument to the class, I can keep a default list and then fetch more items to redact from the config itself. I can then append those items to `secretNamesToRedactInConfigDump` and then carry out the task. I hope this approach is more sustainable to maintain consistency across FRS and FF.

Reply:
Makes sense. As long as you include a way to add to the list on the fly (via config in FRS), we are good.